### PR TITLE
Add martins comment

### PR DESCRIPTION
**Pull request by LukeWood**
@martin-gorner

### CODE REVIEW

**Review by qlzh727 — APPROVED** [review]

File: keras_cv/models/resnet_v2.py [hunk 1]
```diff
@@ -389,6 +389,9 @@ def as_backbone(self):
 
         The output of the backbone model will be a dict with int as key and tensor as
```
Comment (martin-gorner) [inline keras_cv/models/resnet_v2.py:390]: Are you sure it's a dict ? I see a list when I test ([repro colab](https://colab.research.google.com/drive/1-Fvf4sd0X0R14WNeC5899hho9B6hVHEG#scrollTo=Ezw0mHdT3tmy))

File: keras_cv/models/resnet_v2.py [hunk 2]
```diff
@@ -389,6 +389,9 @@ def as_backbone(self):
...
         value. The int key represent the level of the feature output.
+        A typical feature pyramid has five levels corresponding to scales P3, P4, P5,
+        P6, P7 in the backbone. Scale Pn represents a feature map 2n times smaller in
```
Comment (martin-gorner) [inline keras_cv/models/resnet_v2.py:393]: It's not 2n but 2^n (two to the power of n)

File: keras_cv/models/resnet_v2.py (hunk 1 above)
Comment (qlzh727) [reply keras_cv/models/resnet_v2.py:390]: It is a dict, the model.outputs flatten any structure into a dict. If you check model.output, then you will see the dict. You can also check the raw output if you give it a input tensor.

**Pull request merged**